using_images/db_images/postgresql.adoc

```diff
@@ -174,46 +182,75 @@ method to change passwords for the database user (`*POSTGRESQL_USER*`) and
 *postgres* administrator user is by changing the environment variables
 `*POSTGRESQL_PASSWORD*` and `*POSTGRESQL_ADMIN_PASSWORD*`, respectively.
 
+You can view the current passwords by viewing the pod in the web console or by
+listing the environment variables:
+
+----
+$ oc env pod/<pod_name> --list
+----
+
 Changing database passwords through SQL statements or any way other than through
 the environment variables aforementioned will cause a mismatch between the
 values stored in the variables and the actual passwords. Whenever a database
 container starts, it resets the passwords to the values stored in the
 environment variables.
 
-You can change these passwords with:
+To change these passwords, update one or both of the desired environment
+variables for the related deployment configuration(s) using the `oc env`
+command. If multiple deployment configurations utilize these environment
+variables, for example in the case of an application created from a template,
+you must update the variables on each deployment configuration so that the
+passwords are in sync everywhere. This can be done all in the same command:
 
 ----
-$ oc env dc postgresql \
+$ oc env dc <dc_name> [<dc_name_2> ...] \
   POSTGRESQL_PASSWORD=<new_password> \
   POSTGRESQL_ADMIN_PASSWORD=<new_admin_password>
```

Comment:
As you noted in your solution, the variable names are not necessarily the same in the database deploymentconfig and in the other parts of your app (like rails, django, etc).
I think for now this is rather something we need to document rather than addressing case by case.
The name difference is the general case, and we'll be working on making this better when we work on linking services https://trello.com/c/y30pQOUE/688-20-service-deploymentconfig-linking-r-d.